ios/InstagramStories.m

```diff
@@ -82,17 +82,36 @@ - (void)backgroundImage:(NSData *)backgroundImage stickerImage:(NSData *)sticker
     }
 }
 
+- (void)backgroundVideo:(NSData *)backgroundVideo
+{
+    // Verify app can open custom URL scheme. If able,
+    // assign assets to pasteboard, open scheme.
+    NSURL *urlScheme = [NSURL URLWithString:@"instagram-stories://share"];
+    if ([[UIApplication sharedApplication] canOpenURL:urlScheme]) {
+        // Assign background image asset and attribution link URL to pasteboard
+        NSArray *pasteboardItems = @[@{@"com.instagram.sharedSticker.backgroundVideo" : backgroundVideo}];
+        //NSArray *pasteboardItems = @[@{@"com.instagram.sharedSticker.backgroundVideo" : backgroundVideo;
+        NSDictionary *pasteboardOptions = @{UIPasteboardOptionExpirationDate : [[NSDate date] dateByAddingTimeInterval:60 * 5]};
+        // This call is iOS 10+, can use 'setItems' depending on what versions you support
+        [[UIPasteboard generalPasteboard] setItems:pasteboardItems options:pasteboardOptions]; [[UIApplication sharedApplication] openURL:urlScheme options:@{} completionHandler:nil];
+    } else { // Handle older app versions or app not installed case
+        [self fallbackInstagram];
+    }
+}
+
 
 - (void)shareSingle:(NSDictionary *)options
     failureCallback:(RCTResponseErrorBlock)failureCallback
     successCallback:(RCTResponseSenderBlock)successCallback {
     
     NSString *attrURL = [RCTConvert NSString:options[@"attributionURL"]];
+    //NSData *backgroundVideo = [[NSFileManager defaultManager] contentsAtPath:path];
     if (attrURL == nil) {
         attrURL = @"";
     }
     
     NSString *method = [RCTConvert NSString:options[@"method"]];
+       RCTLog(@"method method is");
```

Comment:
can we remove this?

Reply:
Of Course! I 'll remove and make a new request! Thanks.